modules/monitoring-configuring-a-persistent-volume-claim.adoc

```diff
@@ -141,11 +141,9 @@ data:
 Storage requirements for the `thanosRuler` component depend on the number of rules that are evaluated and how many samples each rule generates.
 ====
 
-. Save the file to apply the changes. The pods affected by the new configuration are restarted automatically and the new storage configuration is applied.
-ifdef::openshift-dedicated,openshift-rosa[]
+. Save the file to apply the changes. The pods affected by the new configuration are automatically redeployed and the new storage configuration is applied.
 +
 [WARNING]
 ====
-When changes are saved to a monitoring config map, the pods and other resources in the related project might be redeployed. The running monitoring processes in that project might also be restarted.
+When you update the config map with a PVC confiuration, the affected `StatefulSet` object is recreated, resulting in a temporary service outage."
```

Comment:
@juzhao Please correct me if this is wrong, but when I tested this, the statefulset got redeployed in the same way it does when I resized the PV.  So I included the same warning here as in the resizing section.

Reply:
thanks, you are right